source/common/upstream/cds_subscription.cc

```diff
@@ -27,6 +27,11 @@ void CdsSubscription::createRequest(Http::Message& request) {
   request.headers().insertMethod().value().setReference(Http::Headers::get().MethodValues.Get);
   request.headers().insertPath().value(
       fmt::format("/v1/clusters/{}/{}", local_info_.clusterName(), local_info_.nodeName()));
+  request.headers().insertContentType().value().setReference(
+      Http::Headers::get().ContentTypeValues.Json);
+
+  const size_t empty_body_size = 0;
```

Comment:
nit: I don't think the local var here adds much. 0 is pretty obvious and I would just pass to value. Same elsewhere.